tasks/coverage/parser_babel.snap

```diff
@@ -6833,8 +6833,8 @@ Expect to Parse: "typescript/types/const-type-parameters-babel-7/input.ts"
    ╭─[esprima/es2015-arrow-function/complex-rest-in-arrow-not-allowed/input.js:1:1]
  1 │ (a,...[a]) => 0;
    ·  ┬     ┬
-   ·  │     ╰── It can not be redeclared here
-   ·  ╰── `a` has already been declared here
+   ·  │     ╰── `a` has already been declared here
```

Comment:
Why is the order reversed? Is this easy to fix?

Reply:
Looking at the visit order, we'll go to `FormalParameters` first followed by `FormalParameter`, which causes the order to change, the fix for this I've thought of so far is to execute the `FormalParameters`' binder method during the `leave_kind` phase, but I'm not sure if that will cause any other problems